.travis.yml

```diff
@@ -1,5 +1,4 @@
 language: node_js
 node_js:
   - "0.12"
-  - "0.11"
-  - "0.10"
+  - "5"
```

Comment:
"5" is the latest major release on the list at https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Choosing-Node-versions-to-test-against